config/contributors_schema.toml

```diff
@@ -1,16 +1,6 @@
-[snapshot_id]
-Description = "nan"
-Deduced_Data_Type = "nan"
-Nullable = "nan"
-Current_Data_Type = "str"
-Length = "nan"
-Min_values = "nan"
-Max_values = "nan"
-Possible_Categorical_Values = ["nan"]
-
 [reference]
 Description = "nan"
-Deduced_Data_Type = "int"
+Deduced_Data_Type = "int64"
```

Comment:
2023-08-01 11:18:25,388 - src.staging.validation - validate_data_with_schema - DEBUG:reference before: object
2023-08-01 11:18:25,390 - src.staging.validation - validate_data_with_schema - DEBUG:reference after: object

It changes from object to object even though the deduced data type is int64. No warning was given. Is this expected?